selenium-system-test/src/test/resources/features/SCR Manual Account Creation.feature

```diff
@@ -1,16 +1,20 @@
 @test
 Feature: SCR Manual Account Creation
 
-  Scenario: Call handler creates account
-    Given the operator is logged in with SCR clearance
+  Scenario Outline: Call handler creates account
+    Given the operator is logged with SCR clearance <role>
```

Comment:
I think leaving the 'in' in works better

Reply:
I had to remove it as "the operator is logged with SCR clearance <role>" and "the operator is logged in with <role>" - cucumber is getting confused & failing

Reply:
ok